edge-functions/bot-protection-botd/lib/botd/index.ts

```diff
@@ -33,6 +34,7 @@ export async function botdEdge(
 
   const { pathname } = req.nextUrl
 
+  // Light bot detection is not required for static files or for favicon.ico
   if (STATIC_REGEX_EXCLUSION.test(pathname) && !isFavicon(req)) return
```

Comment:
If the matcher is using `/` and `/blocked` I think it won't execute for static files right?

Reply:
It won't, but I don't think we need to remove this check, might be helpful for people implementing it in their own middleware.